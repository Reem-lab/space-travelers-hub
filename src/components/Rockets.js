const Rockets = () => (
  <h2>Hello from rockets</h2>
);

export default Rockets;
