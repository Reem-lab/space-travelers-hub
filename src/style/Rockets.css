.container {
  width: 100vw;
  height: 100px;
  margin: 20px;
}

.panel {
  display: flex;
  gap: 10px;
  margin-bottom: 20px;
}

.img {
  width: 255px;
}

.info {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
}

.info h2 {
  margin-top: 0;
}

.para {
  max-width: 95%;
  margin: 0;
  padding: 0;
  margin-bottom: 20px;
  text-align: start;
}

.reserveBtn {
  width: 11.5rem;
  height: 2.813rem;
  border-radius: 0.4rem;
  background-color: #0290ff;
  color: #fff;
  font-size: 15px;
  cursor: pointer;
}

.cancelBtn {
  width: 11.5rem;
  height: 2.813rem;
  border-radius: 0.4rem;
  background-color: #fff;
  color: black;
  font-size: 15px;
  cursor: pointer;
}

.reserved {
  background-color: #00d7c6;
  border-radius: 6px;
  width: 13px;
  font-weight: bold;
  padding: 3px;
  margin-right: 9px;
}
