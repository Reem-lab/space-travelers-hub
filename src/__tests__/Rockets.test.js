/* eslint-disable no-unused-vars */
import { Provider } from 'react-redux';
import { render, screen } from '@testing-library/react';
import Rockets from '../components/Rockets';
import store from '../redux/store';

describe('Test the Rockets list', () => {
  it('renders correctly', () => {
    const rocket = [
      {
        id: '12',
        rocket_name: 'falcony',
        description: 'The Falcon 1 was an expendable launch system privately developed and manufactured by SpaceX during 2006-2009. On 28 September 2008, Falcon 1 became the first privately-developed liquid-fuel launch vehicle to go into orbit around the Earth',
      },
    ];

    const { container } = render(
      <Provider store={store}>
        <Rockets />
      </Provider>,
    );
    const message = screen.getByText('No Rockets Found ❗❕');
    expect(message).toMatchSnapshot();
  });
});
