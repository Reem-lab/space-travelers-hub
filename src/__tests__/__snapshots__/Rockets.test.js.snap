// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Rockets list renders correctly 1`] = `
<div
  class="heading"
>
  No Rockets Found ❗❕
</div>
`;
